applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -957,7 +962,8 @@ void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref
 	int ret;
 	size_t pcm_size;
 
-	ret = sw_codec_decode(buf, size, bad_frame, &ctrl_blk.decoded_data, &pcm_size);
+	ret = sw_codec_decode(buf->data, buf->len, audio_frame->meta.bad_data,
```

Comment:
You may have "saved this for later", I think `sw_codec_decode` should just take `(struct audio_data *audio_frame)`

Reply:
Correct, it was saved